lib/src/over_react_test/common_component_util.dart

```diff
@@ -531,7 +530,7 @@ void testRequiredProps(BuilderOnlyUiFactory factory, dynamic childrenFactory())
     isComponent2 = version == ReactDartComponentVersion.component2;
 
     var jacket = mount(factory()(childrenFactory()), autoTearDown: false);
-    var consumedProps = (jacket.getDartInstance() as component_base.UiComponent).consumedProps;
+    var consumedProps = (jacket.getDartInstance() as component_base.UiComponent).consumedProps ?? [];
```

Comment:
I think due to the way this is overridden in the generated class, we can assume this is non-null. Plus, it was non-null before, otherwise the for-loop would have thrown.
```suggestion
    var consumedProps = (jacket.getDartInstance() as component_base.UiComponent).consumedProps;
```